networking/igmp/conformance/common/service.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# This is basic include file
+
+lib_service()
+{
+	local service=$1
+	local action=$2
+	if [ "$(GetDistroRelease)" -ge 7 ];then
+		systemctl $action ${service}.service
+	else
+		service $service $action
+	fi
+}
+
+# TODO: save iptales on RHEL7, then iptables -F
```

Comment:
Is the todo needed for proper test functionality?

Reply:
I think should be ok to merge without implementing it on my understanding,

@rasibley,  do you have comment and thought?

Reply:
Yeah, we see this a lot, the ToDo may no longer be needed or they haven't had time update the test, it runs without issues downstream and should be ok to port as is, as long as it passes continue to pass with upstream kernels :-)